R/teal_slice.R

```diff
@@ -486,3 +489,52 @@ slices_which <- function(tss, expr) {
   expr <- str2lang(expr)
   Filter(function(x) isTRUE(eval(expr, x)), tss)
 }
+
+
+# sets selected fields to NULL in all `teal_slice`s in a `teal_slices`
+#' @rdname teal_slice
+#' @keywords internal
+#'
+slices_drop <- function(tss, drop = NULL) {
```

Comment:
drop argument isn't used in the code base
```suggestion
slices_drop <- function(tss) {
```

Reply:
It is in unit tests.